graph.py

```diff
@@ -2,13 +2,36 @@
 import json
 from osbs.utils import strip_registry_from_image
 import sys
+import datetime
+import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+import re
+
+
+def sizeof_fmt(num, suffix='B'):
+    for unit in ['', 'K', 'M', 'G', 'T', 'P', 'E', 'Z']:
+        if abs(num) < 1024.0:
+            return "%3.1f%s%s" % (num, unit, suffix)
+        num /= 1024.0
+    return "%.1f%s%s" % (num, 'Yi', suffix)
 
 
 class BuildTree(object):
-    def __init__(self, builds):
+    def __init__(self, builds, pulp_base_url):
```

Comment:
Yes, but the results would be incorrect - since some builds have compressed size stored and some don't.

I'll probably update the script for OSD - both image-id and size should be correct there